### PR TITLE
Add an initial issue template based on React's template

### LINKED ISSUE #1775: Create an issue template for Redux (and other ReactJS projects?)
I'm seeing an awful lot of issues being filed that are either usage questions or not actually Redux-related issues (boilerplate problems, etc).  Per https://reactcommunity.org/ , the ReactJS org policy is: 

> 3) Avoid Burnout

> The first priority in managing the issue tracker for a reactjs project is avoiding burnout. Sprawling lists of inactionable, open issues, is burnout’s most potent fuel.

> This means:
> - Bug reports without test cases (copy/paste-able code is enough) can be closed with or without comment.
> - Usage questions can be closed immediately and directed to stack overflow. Owners should pay attention though, because the docs or the APIs probably need help if there are a lot of “question issues”.
> - Feature requests without thoughtful commentary and sample code can be closed with or without comment.
> - Issues without activity in the last 30 days can be closed with or without comment. If this is a bug you care about that isn’t getting attention, fix it. If you’re good enough to understand the bug, you’re good enough to fix it. Remember, it’s our code now.

Based on that, I'd say we ought to create an issue template for Redux that expresses these ideas, to try to cut down on non-bug issues, and redirect people to SO or Reactiflux for questions.  

Do any other ReactJS projects have an issue template already?  If not, should we make a common one for use in multiple repos?

### PR DESCRIPTION
**Pull request by markerikson**
Fixes #1775

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+**Do you want to request a *feature* or report a *bug*?**  
+(If this is a *usage question*, please **do not post it here** - post it on [Stack Overflow](http://stackoverflow.com/questions/tagged/redux) instead. If this is not a "feature" or a "bug", or the phrase "How do I...?" applies, then it's probably a usage question.)
```
Comment (gaearon) [inline .github/ISSUE_TEMPLATE.md]: Nit: “ ” —

Comment (markerikson) [inline .github/ISSUE_TEMPLATE.md]: Aw, man, your typography is _killing_ me :)

Comment (markerikson) [inline .github/ISSUE_TEMPLATE.md]: Actually, I'd point out that this isn't even part of the docs - it's just an issue template.

Comment (gaearon) [inline .github/ISSUE_TEMPLATE.md]: It’s still in the repo 😄 . I can fix it myself.

Comment (markerikson) [inline .github/ISSUE_TEMPLATE.md]: Heh.  Okay - I'll merge if you're good with it otherwise.

Comment (gaearon) [inline .github/ISSUE_TEMPLATE.md]: Sure

**Pull request merged**